draft-ietf-quic-transport.md

```diff
@@ -1079,6 +1079,10 @@ deployment-specific) method that will allow packets with that connection ID to
 be routed back to the endpoint and to be identified by the endpoint upon
 receipt.
 
+Multiple connection IDs are used so that endpoints can send packets that cannot
+be identified as being for the same connection without cooperation from an
+endpoint; see {{migration-linkability}}.
```

Comment:
```suggestion
be identified as being for the same connection without cooperation from their
peer; see {{migration-linkability}}.
```

Reply:
This is talking about any entity doing the linking, and either endpoint in the connection can provide them with the ability to correlate packets for the same connection.  What I proposed is, I believe, correct.